<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>声光调制器任意波形控制器的上位机软件</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="mainGridLayout">
    <item row="1" column="7">
     <widget class="QPushButton" name="pushButtonStart">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>开始调制</string>
      </property>
     </widget>
    </item>
    <item row="2" column="7">
     <widget class="QPushButton" name="pushButtonExit">
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QPushButton" name="pushButtonConnect">
      <property name="text">
       <string>连接下位机</string>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QPushButton" name="pushButtonSend">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>发送波形</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6" colspan="2">
     <widget class="QGroupBox" name="groupBoxStatus">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>状态</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEditFrequencyStatus">
         <property name="text">
          <string>N/A</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditConnectStatus">
         <property name="text">
          <string>未连接</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelVoltageStatus">
         <property name="text">
          <string>电压量化位数：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelFrequencyStatus">
         <property name="text">
          <string>调制频率：</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEditVoltageStatus">
         <property name="text">
          <string>N/A</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelConnectStatus">
         <property name="text">
          <string>下位机连接状态：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelPort">
         <property name="text">
          <string>端口：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBoxPort"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2" rowspan="3" colspan="4">
     <widget class="QGroupBox" name="groupBoxWave">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="title">
       <string>波形预览</string>
      </property>
      <layout class="QGridLayout" name="gridLayout" columnstretch="2,5">
       <item row="0" column="0">
        <widget class="QLabel" name="labelModulatingWave">
         <property name="text">
          <string>正在调制波形：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCustomPlot" name="widgetEditingWave" native="true"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelEditingWave">
         <property name="text">
          <string>正在编辑波形：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCustomPlot" name="widgetModulatingWave" native="true"/>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QPushButton" name="pushButtonEditWave">
         <property name="text">
          <string>编辑波形</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="actionEditWave"/>
    <addaction name="actionSetting"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionEditWave">
   <property name="text">
    <string>波形编辑器</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="text">
    <string>参数设置</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonExit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>790</x>
     <y>565</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEditWave</sender>
   <signal>triggered()</signal>
   <receiver>pushButtonEditWave</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
